pkg/collector/corechecks/snmp/checkconfig/config_metric.go

```diff
@@ -199,22 +145,6 @@ func normalizeRegexReplaceValue(val string) string {
 	return re.ReplaceAllString(val, "$$$1")
 }
 
-// transformIndex change a source index into a new index using a list of transform rules.
-// A transform rule has start/end fields, it is used to extract a subset of the source index.
-func transformIndex(indexes []string, transformRules []MetricIndexTransform) []string {
```

Comment:
note: moved without content change to pkg/collector/corechecks/snmp/report/report_utils.go